src/test/unit/InboundServer.test.js

```diff
@@ -11,7 +11,8 @@
 'use strict';
 
 const util = require('util');
-const Readable = require('stream').Readable;
+const DummyRequest = require('./dummyRequest.js');
```

Comment:
I think `.js` suffix is redundant here.
Also, I think that it will be better to use Pascal case for filenames containing types (`DummyRequest.js`).

Reply:
facepalm. sigh. if you insist.